.dockerignore

```diff
@@ -1 +1,3 @@
 client/node_modules
+# simple hello world image for testing
+local
```

Comment:
what is this for?

Reply:
I used a local docker-file in the project itself to test out my work. Did not plan on pushing it into the repo or the docker image, so added it to the ignore files. 

This can also serve as a temporary storage for the project. It is not pushed into git repo or the image, so anything here is just on the FS and not anywhere else. Helped me test the project without messing around and also make sure that the file path was known to be if anything went wrong.

Reply:
Do you mind removing it? Just don't stage those files when committing. We cannot adjust the ignore files to all the non-standard setups that everybody contributing to this project has. Thanks!